staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec_test.go

```diff
@@ -980,6 +980,28 @@ func TestRefreshCreds(t *testing.T) {
 			}`,
 			wantCreds: credentials{token: "foo-bar"},
 		},
+		{
+			name: "v1-basic-request",
+			config: api.ExecConfig{
+				APIVersion:      "client.authentication.k8s.io/v1",
+				InteractiveMode: api.IfAvailableExecInteractiveMode,
```

Comment:
Failing test case for when this is not specified?

Reply:
https://github.com/kubernetes/kubernetes/pull/102890/commits/2ad77fc40667bfba7cada0b7d8b79a8f37ec477f